README.md

```diff
@@ -316,6 +316,9 @@ julia> filter(f -> endswith(f, ".log"), readdir(pwd()))
 ```
 
 The user implicitly controls when the files will be rolled over based on the `DateFormat` given.
+The `DatetimeRotatingFileLogger` is backed by a `SimpleLogger` by default, but it can also
+be backed by a `FormatLogger` by passing in message formatting function (see below for the
+required format) as the first argument in the constructor.
```

Comment:
See my comment below about maybe we should make this an implementation detail and not promise anythign about how it works, just what it does

Reply:
Makes sense, will update.